src/Storage/Controllers/SignController.cs

```diff
@@ -35,6 +41,7 @@ public SignController()
         [Produces("application/json")]
         public async Task<ActionResult> Sign([FromRoute] int instanceOwnerPartyId, [FromRoute] Guid instanceGuid, [FromBody] SignRequest signRequest)
         {
+            await _instanceService.CreateSignDocument(instanceOwnerPartyId, instanceGuid, signRequest);
```

Comment:
Should we return the signing document?

Reply:
We discussed this friely, but didn't conclude with anything. Service is still being developed by Thomas so output can be updated. Simply cherry picked the files I needed from his dev branch to be able to complete implementation of the event logic